### ISSUE TITLE
Limited to 100TB storage

### ISSUE THREAD

**Issue by zenjabba** [yadayada/acd_cli#370]
ACD_CLI reports to FUSE 100TB storage, so once you go above that amount, it will report "out of space".

Can we report 1PB of free space?

**Comment by charlymr** [comment]
Amazon will ask you question before you reach that I would say... 
It is unlimited with (fair usage policy)... Are you close to that amount of space use already? Just curious as if Amazon did tell you anything?

**Comment by zenjabba** [comment]
yes I am close to that amount, and amazon has not questioned my amount stored in my two accounts.

**Comment by charlymr** [comment]
Fairplay 👍

**Comment by jrwr** [comment]
I think thats the endpoint data from amazon is the quota side, if you look in .cache/endpoint_cache it says the 100TB max size

I think this might be Unlimited* on ACD's part

**Comment by asabla** [comment]
I've noticed this as well. Could you please comment here again @zenjabba if you get past 100TB with the result?

**Comment by zenjabba** [comment]
So I've looked at other systems, and they also report 100TB as the storage space available, but for example NetDrive reports

100TB of 100TB Free
<img width="273" alt="screen shot 2016-08-05 at 8 02 14 pm" src="https://cloud.githubusercontent.com/assets/679864/17453292/99533e40-5b47-11e6-95c5-b70a3cbf8604.png">
 so I don't feel it's a limitation within amazon.

**Comment by bryan** [comment]
It appears to be a 100TB soft cap, where you can call Amazon to allow for more space (probably have to give them a good reason given that 100TB is quite a bit of storage). Not certain though since I can't seem to find anyone hitting the 100TB cap. Keep us in the loop! 

https://github.com/dularion/streama/issues/63

**Comment by roaima** [comment]
If it's any help, S3QL (another cloud filesystem) reports double the usage with a minimum of 1TB, so you never exceed 50℅ of the reported space.

**Comment by zenjabba** [comment]
The problem is S3QL doesn’t support ACD, so apples and oranges (last time I looked at S3QL it didn’t)

**Comment by yadayada** [comment]
Does the reported disk size really have any practical implications?

**Comment by zenjabba** [comment]
yes, when you try to upload a file and the disk reports 100% usage, you get a error

**Comment by tristaoeast** [comment]
@zenjabba  did you try uploading using rclone? I'm just curious to see if it presents the same limitation. I'm currently using acdcli to mount my ACD and rclone to upload to it 

(sorry @yadayada for PRing the competition, it's just that I'm more familiared with the rsync syntax -- although you have some interesting options in your upload command that I still ought to try)

**Comment by zenjabba** [comment]
So finally got a chance to get back to this

ACDFuse        107374182400 -9444732965617890843136 -14025401856 100% /mnt/amazon 

is what is reported when it's over 100GB. Can we please just get it to report 1PB of storage so it will never run out.

Thanks
